_projects/2023/100516517/100516517.Rmd

```diff
@@ -0,0 +1,798 @@
+---
+title: "Final Project - Candela Gómez Blanco"
+description: |
+  A short description of the post.
```

Comment:
Please add a descriptive title and a short abstract with a bit of context about the graph chosen.